nixos/modules/config/shells-environment.nix

```diff
@@ -157,6 +157,8 @@ in
     # terminal instead of logging out of X11).
     environment.variables = config.environment.sessionVariables;
 
+    environment.profileRelativeEnvVars = config.environment.profileRelativeSessionVariables;
```

Comment:
I thought this seemed unnecesssary, then I read the comment on the assignment above.  Maybe move this up a line so it obviously belongs with the previous one.